apps/acrs/views/start.htm

```diff
@@ -0,0 +1,6 @@
+{{<partials-page}}
+  {{$page-content}}
+    
+    {{#input-submit}}continue-only{{/input-submit}}
+  {{/page-content}}
+{{/partials-page}}
```

Comment:
Jake has just updated the /start page content in Figma - its a bit different from this now.

Reply:
Thanks i will update that